src/main/resources/processes/FOI_ACCEPTANCE.bpmn

```diff
@@ -9,14 +9,15 @@
       <bpmn:outgoing>Flow_0eg5k1f</bpmn:outgoing>
```

Comment:
About ${execution.getVariable('DraftAcceptCase') != null && DraftAcceptCase == "N"}.
Not sure that DraftAcceptCase is the right variable to use - it is the output of a user task in a future stage. What we need to check is has this case already been accepted by the acceptance team. So I think maybe we should set a variable (date accepted?) in this stage when a case is accepted, and test for the presence of that variable.

Reply:
I have made this change